ztp/gitops-subscriptions/argocd/example/policygentemplates/hub-side-templating/template-group-du-sno-ranGen.yaml

```diff
@@ -0,0 +1,254 @@
+        accessModes:
+          - ReadWriteMany
+        resources:
+          requests:
+            storage: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-storagepvc-storage" (index .ManagedClusterLabels "group-du-sno-zone")) hub}}'
+        storageClassName: image-registry-sc # FIXED
+        volumeMode: Filesystem # FIXED
+    #############
+    # pv-policy #
+    #############
+    # The following usage of ImageRegistryPV.yaml is assuming that mount_point is set to `/var/imageregistry` in SiteConfig
+    # using StorageClass `image-registry-sc` (see the first sc-file)
+    - fileName: ImageRegistryPV.yaml  # wave 2
+      policyName: "pv-policy"
+    #############
+    # irc-policy #
+    #############
+    - fileName: ImageRegistryConfig.yaml # wave 11
+      policyName: "irc-policy"
+      spec:
+        storage:
+          pvc:
+            claim: "image-registry-pvc" # FIXED
+    # ---- END OF source CRs needed for image registry (check ImageRegistry.md for more details) ----
+    ################
+    # sriov-policy #
+    ################
+    - fileName: SriovNetwork.yaml # wave 100
+      policyName: "sriov-policy"
+      metadata:
+        name: sriov-nw-du-fh # FIXED
+      spec:
+        resourceName: du_fh # FIXED
+        vlan: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-sriov-network-vlan-1" .ManagedClusterName) | toInt hub}}'
+        ipam: |
+          {
+            "type": "host-local",
+            "subnet": '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-sriov-network-ipam-subnet-1" .ManagedClusterName) hub}}',
+            "rangeStart": "10.56.217.171",
+            "rangeEnd": "10.56.217.181",
+            "routes": [{
+              "dst": "0.0.0.0/0"
+            }],
+            "gateway": "10.56.217.1"
+          }
+    ###
+    - fileName: SriovNetworkNodePolicy.yaml # wave 100
+      policyName: "sriov-policy"
+      metadata:
+        name: sriov-nnp-du-fh
+      spec:
+        deviceType: netdevice # FIXED
+        isRdma: false
+        nicSelector:
+          pfNames: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-sriov-node-policy-pfNames-1" (index .ManagedClusterLabels "hardware-type")) | toLiteral hub}}'
+        numVfs: 8
+        priority: 10
+        resourceName: du_fh
+    ###
+    - fileName: SriovNetwork.yaml # wave 100
+      policyName: "sriov-policy"
+      metadata:
+        name: sriov-nw-du-mh # FIXED
+      spec:
+        resourceName: du_mh # FIXED
+        vlan: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-sriov-network-vlan-2" .ManagedClusterName) | toInt hub}}'
+        ipam: |
+          {
+            "type": "host-local",
+            "subnet": '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-sriov-network-ipam-subnet-2" .ManagedClusterName) hub}}',
```

Comment:
Maybe this would be more suitable for a zone label rather than .ManagedClusterName